pkg/datapath/loader/loader.go

```diff
@@ -346,19 +346,21 @@ func (l *Loader) reloadDatapath(ctx context.Context, ep datapath.Endpoint, dirs
 	return nil
 }
 
-func (l *Loader) replaceNetworkDatapath(ctx context.Context, interfaces []string) error {
+func (l *Loader) replaceNetworkDatapath(ctx context.Context, interfaces []string) (err error) {
 	progs := []progDefinition{{progName: symbolFromNetwork, direction: dirIngress}}
 	for _, iface := range option.Config.EncryptInterface {
-		finalize, err := replaceDatapath(ctx, iface, networkObj, progs, "")
-		if err != nil {
-			log.WithField(logfields.Interface, iface).WithError(err).Fatal("Load encryption network failed")
+		finalize, replaceErr := replaceDatapath(ctx, iface, networkObj, progs, "")
+		if replaceErr != nil {
+			log.WithField(logfields.Interface, iface).WithError(replaceErr).Error("Load encryption network failed")
+			// Return the error to the caller, but keep trying replacing other interfaces.
+			err = replaceErr
```

Comment:
Could such an error ever happen during [the first IPsec initialization](https://github.com/cilium/cilium/blob/8582a62760c3cb1cb496bceb319b4fc679fa9b1b/pkg/datapath/loader/base.go#L507)? If it does, then we will skip a bunch of things. Is that okay? Will Reinitialize retry? Even if it retries, it does mean the watcher for netlink interfaces will never be started.